packages/react/src/codemods/codemods-helpers.tsx

```diff
@@ -104,16 +104,33 @@ export const findImportSpecifierName = ({
   return getImportSpecifierName(importSpecifierCollection);
 };
 
+const buildImportDeclaration = ({
+  j,
+  namedImport,
+  localName,
+}: {
+  j: JSCodeshift;
+  namedImport: string;
+  localName: string;
+}): ImportDeclaration[] => [
+  j.importDeclaration(
+    [j.importSpecifier(j.identifier(namedImport), j.identifier(localName))],
+    j.literal(COMPILED_IMPORT_PATH)
+  ),
+];
+
 export const convertDefaultImportToNamedImport = ({
   j,
   collection,
   importPath,
   namedImport,
+  featureFlagExpression,
 }: {
   j: JSCodeshift;
   collection: Collection<any>;
   importPath: string;
   namedImport: string;
+  featureFlagExpression: string | null;
```

Comment:
Could this become more generic, so that we don't necessarily have to add in more options later down the line? I'm thinking of a plugin system specifically, where compiled exposes this feature flag implementation you already have as a plugin. Then products should create their own plugins for they import the flag, and possibly include a script to generate a separate file (e.g. `feature-flags.js` in Jira).

Reply:
Sounds good! I'll spike out a quick plugin system in a different PR